### PR TITLE
ToJavascript instance of String and Text

### PR DESCRIPTION
**Pull request by pythonissam**

### CODE REVIEW

**Comment by pythonissam** [comment]
issue ref: https://github.com/yesodweb/shakespeare/issues/226

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+### 2.0.18
+
+* Text instance of String and Text [#227](https://github.com/yesodweb/shakespeare/pull/227)
```
Comment (snoyberg) [inline ChangeLog.md]: I think this should say ToJavascript instance, and point out that it's for both strict and lazy Text.

Comment (psibi) [reply ChangeLog.md]: I think it should say `ToJavascript instance for String, Strict and Lazy Text`. Right now, it says `Text instance for ....` instead.

**Comment by snoyberg** [comment]
Thanks!

**Pull request merged**